tools/jcw-gen/App.cs

```diff
@@ -81,13 +85,15 @@ public static int Main (string [] args)
 			}
 		}
 
-		static void GenerateJavaCallableWrapper (TypeDefinition type, string outputPath, TypeDefinitionCache cache)
+		static void GenerateJavaCallableWrapper (TypeDefinition type, string outputPath, TypeDefinitionCache cache, JavaPeerStyle style)
 		{
 			if (type.IsInterface) {
 				return;
 			}
+			Console.WriteLine ($"# jonp: jcw-gen: emitting JCW for {type.FullName}");
```

Comment:
Are some of the `#jonp` log messages leftover from debugging?

Reply:
*Lots* of leftovers from debugging. :-)